app/templates/components/task-card.hbs

```diff
@@ -7,15 +7,47 @@
     <span class="task-card__title" data-test-selector="task title">{{task.title}}</span>
     <span class="task-card__number" data-test-selector="task number">#{{task.number}}</span>
   </p>
-  {{related-skills class="task-card__skills" skills=taskSkills}}
+  {{#if taskSkills}}
+    {{related-skills class="task-card__skills" skills=taskSkills}}
+  {{/if}}
   <p class="task-card__meta">
-    <span class="task-card__icon"></span>
     <span class="task-card__time" data-test-selector="task time">{{moment-from-now task.insertedAt}}</span>
   </p>
-  {{task/user/user-select
+  {{#power-select
+    class="select-inline"
+    dropdownClass="select-inline-dropdown"
+    loadingMessage=""
+    matchTriggerWidth=false
+    onchange=(action "selectUser")
+    onselect=(action "selectUser")
+    options=sortedUserOptions
+    placeholderComponent="task-card/user/unselected-item"
+    registerAPI=(action (mut selectRemoteController))
+    search=(action "searchUsers")
+    selected=selectedOption
+    selectedItemComponent="task-card/user/selected-item"
+    tagName="div"
+    as |user|
+  }}
+    <div class="select-inline-dropdown__list-item__avatar">
+      <img src={{user.photoThumbUrl}} />
+    </div>
+    <div class="select-inline-dropdown__list-item__content">
+      <span class="select-inline-dropdown__list-item__username">{{user.username}}</span>
+      {{#if user.name}}
+        <span class="select-inline-dropdown__list-item__name">{{user.name}}</span>
+      {{else}}
+        <span class="select-inline-dropdown__list-item__name">{{user.username}}</span>
+      {{/if}}
+    </div>
+  {{/power-select}}
+  {{!-- {{task/user/user-select
```

Comment:
Any particular reason why this is commented out but not removed? I see a few related lines of component code that could get removed to in that case.